tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -1694,6 +1694,24 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     return do_rewrite;
   }
 
+  static bool Maxpool3DGradRewrite(const Node* n) {
+    CHECK_NOTNULL(n);
```

Comment:
Nit: We don't use `CHECK*` in new code anymore.
```suggestion
    DCHECK(n);
```

Reply:
I have replaced all "CHECK_NOTNULL" usage with "DCHECK" in the same source file - there are ~20. 

I will push the changes once local test is done.

Reply:
Thank you! Just for future reference, it's not required change `CHECK*` in old code. We just try not to have `CHECK*` in any part that is changed/added by the PR. :)